### PR TITLE
completed the initial project set-up 

### PR DESCRIPTION
**Pull request by OpusLearning**
making a public repo creating the…database, and updating the readme file, install husky

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,9 +1,11 @@
...
 
-For instructions, please head over to [L2C NC News](https://l2c.northcoders.com/courses/be/nc-news).
+Set-up instructions
 
+## To run this
 
+Create .env files .env.test and .env.development in the root of the folder
 
---- 
+Add DB name (check SQL set up)
```
Comment (nc-modou) [inline README.md:9]: Which DB name should go to which file? Where shoudl they look when you mention the SQL set up? It's worth being more explicit here so there isn't any room for confusion for someone trying to run your project locally :)

File: README.md [hunk 2]
```diff
@@ -1,9 +1,11 @@
...
 
-This portfolio project was created as part of a Digital Skills Bootcamp in Software Engineering provided by [Northcoders](https://northcoders.com/)
+install dependencies
```
Comment (nc-modou) [inline README.md:11]: I'd also be explicit about the command needed do to this

**Review by nc-modou — COMMENTED** [review]
Changes look good so far - I'd recommend being a bit more explicit in what steps are needed however

File: README.md (hunk 1 above)
Comment (OpusLearning) [reply README.md:9]: Thanks Modou, i will add some further detail to the readme :)

**Pull request merged**